lib/node_modules/@stdlib/blas/base/daxpy/README.md

```diff
@@ -207,7 +207,7 @@ The function accepts the following arguments:
 -   **X**: `[in] double*` input array.
 -   **strideX**: `[in] CBLAS_INT` index increment for `X`.
 -   **Y**: `[inout] double*` output array.
--   **strideY**: `[in CBLAS_INT` index increment for `Y`.
+-   **strideY**: `[in] CBLAS_INT` index increment for `Y`.
```

Comment:
Thanks for fixing this. In the future, however, we'll want to open a separate PR in order to ensure a clean commit history and changelog.